source/chapters/configuration.rst

```diff
@@ -183,3 +207,304 @@ already use JACK, it is easiest to let Mixxx suspend PulseAudio and use ALSA.
+   :alt: Mixxx preferences - BPM settings
+   :figclass: pretty-figures
+
+   Mixxx preferences - BPM settings
+
+The table below summarizes the beat detection settings:
+
++---------------------------------------+--------------------------------------+
+| Option                                | Description                          |
++=======================================+======================================+
+| Enable Fast Analysis                  | If enabled, BPM  will be detected by |
+|                                       | only using the first minute of audio.|
++---------------------------------------+--------------------------------------+
+| Assume constant tempo                 | If enabled, Mixxx assumes that the   |
+|                                       | distances between the beats are      |
+|                                       | constant. If disabled, the raw beat  |
+|                                       | grid obtained by the analyzer is     |
+|                                       | presented. The latter is appropriate |
+|                                       | for tracks with variable tempo.      |
++---------------------------------------+--------------------------------------+
+| Enable Offset Correction              | Prevents beat markers from being     |
+|                                       | placed incorrectly.                  |
++---------------------------------------+--------------------------------------+
+| Re-analyze beats when settings        | If enabled, Mixxx over-writes old    |
+| change or beat detection data is      | beat grids from Mixxx before v1.11.  |
+| outdated                              | Moreover, it will re-analyze the BPM |
+|                                       | if your beat detection preferences   |
+|                                       | change or BPM data from 3rd party    |
+|                                       | programs are present.                |
++---------------------------------------+--------------------------------------+
+
+Correcting Beat Grids
+---------------------
+
+There may be situations where BPM and beat detection do not result in a proper
+beat grid.
+
+Typically, the detected BPM is correct but the analyzer has failed to detect the
+location of the first beat. Consequently, the beat markers are shifted, i.e.
+the beat markers are always a fixed distance from the true beat. To adjust the
+beat grid, cue the track before a real beat and click the :guilabel:`Beat-grid
+Adjust` button in the :ref:`interface-button-grid`.
+
+If the detected BPM is not accurate, the corresponding beat grid will also be
+inaccurate. A deviation of 0.02 BPM units from the correct BPM will cause
+beatgrid alignment issues on long tracks (e.g. a club mix). If this happens,
+your beatgrid may look aligned for the few minutes but you will notice a slight
+drift as the song goes on. Finding the correct BPM is easy in many cases - just
+follow the note below.
```

Comment:
we can do a lot more adjustments, we have keybindings to tweak offset and speed.  Something else to update later